src/working-mode

```diff
@@ -0,0 +1,252 @@
+  issues<a class="self-link" href="#non-normative-issues"></a></h4>
+
+  <p>Changes of editorial nature can be made, accepted, or rejected by the editor without
+  discussion.
+
+  <h3 id="changes">Changes</h3>
+
+  <p>Each normative change made to the standard needs to meet the following criteria:
+
+  <ul>
+   <li>It must have support from implementers.
+   <li>It should have corresponding test changes, either in the form of new tests or modifications
+   to existing tests.
+   <li>Implementations bugs must be filed for each user agent that fails tests. (This is each user
+   agent that doesn't match the proposed changes. If the test changes are not adequate to reveal
+   that, but it's known through other means, the tests should be improved first.)
+   <li>It should have been reviewed by one or more members of the community.
+   <li>Optional or implementation-defined behavior must be very well motivated.
+  </ul>
+
+  <p>Additionally, the following more mechanical criteria are used to hone a change before accepting
+  it:
+
+  <ul>
+   <li>It should match the standard's source formatting and style conventions. (Consistent
+   formatting helps new contributors know what to do.)
+   <li>It must have a good commit message, with at the minimum a title, but typically with a body
+   that includes pointers to relevant discussion, tests changes, and a more elaborate description of
+   the change. (A detailed commit message helps folks in the future figure out why a decision was
+   made. This happens a lot more than you might imagine.)
+  </ul>
+
+  <p class="note">If the change is proposed by a new contributor, the editor can ensure the commit
+  message is good, and help with formatting and style. There's no need to make new contributors jump
+  through too many hoops.
+
+  <h4 id="additions">Additions<a class="self-link" href="#additions"></a></h4>
+
+  <p>Any change that represents an addition needs to meet these additional criteria:
+
+  <ul>
+   <li>The addition must have the support of at least two implementers. (Such support is not binding
+   in any way on the implementers, however.)
+  </ul>
+
+  <p>Additionally, the following are strongly recommended:
+
+  <ul>
+   <li>The support from implementers should be of the form "we would like to implement this soon"
+   and not just "this seems like a reasonable idea".
+
+   <li>There should be no strong implementer objections to the new feature.
+
+   <li>There should already be a prototype implementation or one being worked on side-by-side with
+   the change to the standard.
+
+   <li>New features should be something that cannot be done today or are such a common trend in
+   libraries that standardizing them helps lots of people. Alternatively, the "new" feature could
+   actually be a legacy feature that was not yet standardized, which is now being standardized. (A
+   legacy "addition" is important to allow for more meaningful competition and interoperability.)
```

Comment:
I read this as "important to allow" as if legacy things were not allowed by some default. Trying a small rewrite.